docker-compose.yml

```diff
@@ -22,7 +21,7 @@ services:
         depends_on:
             - php
         ports:
-            - "${SERVER_HTTP_PORT}:80"
+            - "80:80"
```

Comment:
Please don't use the default ports. Use an environment variable to set the web port — is correct.

Reply:
done